### PR TITLE
Add [Term Entry] Go Math Functions Trunc()

### LINKED ISSUE #3222: [Term Entry] Go Math Functions Trunc()
### Reason/inspiration (optional)

We would like a new term entry for [Go](https://www.codecademy.com/resources/docs/go) about the `Trunc()` function.

The entry should go in a new file in `docs/content/go/concepts/math-functions/terms/trunc/trunc.md`

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```go ` code block. 
- A runnable codebyte example in a ` ```codebyte/golang` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Jamesllllllllll**
### Description

Adding term for function `Trunc()` for Go. Closes #3222 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/math-functions/terms/trunc/trunc.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The **`Trunc()`** function returns the integer value of a given number. This function is part of the `math` package, and the package must be imported to use the function in a Go program.
+
+## Syntax
+
+```pseudo
+result := math.Trunc(x)
+```
+
+Where `result` is the integer value of `x`, returned as a float64 except under the following circumstances:
```
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/trunc/trunc.md:23]:
```suggestion
Where `result` is the integer value of `x`, returned as a `float64` except under the following circumstances:
```

File: content/go/concepts/math-functions/terms/trunc/trunc.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+- The result of `Trunc(math.Inf(1))` is `+Inf`
```
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/go/concepts/math-functions/terms/trunc/trunc.md:25]: update the other lines as above

**Review by caupolicandiaz — APPROVED** [review]
@Jamesllllllllll I've added my comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@Jamesllllllllll This is ready for a second review.

**Review by yangc95 — APPROVED** [review]

**Pull request merged**